test/functional/fixtures/reporter/testcafe-fixtures/skipped-tests.js

```diff
@@ -0,0 +1,22 @@
+fixture `Skipped tests`
+    .page `http://localhost:3000/fixtures/reporter/pages/index.html`;
+test('Simple test', async t => {
+    await t.wait(1);
+    await t.report();
+});
+test.skip('Simple command err test', async t => {
+    await t.click('#non-existing-target');
+});
+
+test.skip('Complex command test', async () => {
```

Comment:
I guess you copy tests from somewhere. The `Complex command test` is a strange name for the empty test. Rename the empty tests as `empty test 1`, `empty test 2`, etc.